src/events.psm1

```diff
@@ -468,7 +468,7 @@ function New-SafeguardEventSubscription
         if ($PSBoundParameters.ContainsKey("UserToSubscribe"))
         {
             $local:UserEmailAddress = (Get-SafeguardUser -AccessToken $AccessToken -Appliance $Appliance -Insecure:$Insecure -UserToGet $UserToSubscribe).EmailAddress
```

Comment:
We can actually get rid of this line. The intention here is to check if the provided User has an email address configured or not. I am initializing the "User" variable in Line 439, which I believe I added later and forgot to get rid of this line. Let me test it to make sure removing this doesn't break anything.